app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -33,15 +33,15 @@ private[mappings] class LocalDateFormatter(
   val log: LoggerLike = Logger(this.getClass)
   val currentDate: LocalDate = LocalDateTime.now().toLocalDate
 
-  private def getEndDay(month: Int, year: Int, date: LocalDate): Int = {
-    if (month == currentDate.getMonthValue && year == currentDate.getYear) {
-      log.info("entered month: " + month + "; entered year: " + year)
-      log.info("current month: " + currentDate.getMonthValue + "; currentyear: " + currentDate.getYear)
-      currentDate.getDayOfMonth
-    } else {
-      date.lengthOfMonth()
-    }
-  }
+//  TODO private def getEndDay(month: Int, year: Int, date: LocalDate): Int = {
```

Comment:
If you want to keep this method for that reason, better to do a block comment and add TODO (with proper msg as below)

/* TODO: May need to uncomment in future
private def getEndDay(month: Int, year: Int, date: LocalDate): Int = {
 if (month == currentDate.getMonthValue && year == currentDate.getYear) {
      log.info("entered month: " + month + "; entered year: " + year)
      log.info("current month: " + currentDate.getMonthValue + "; currentyear: " + currentDate.getYear)
      currentDate.getDayOfMonth
    } else {
      date.lengthOfMonth()
   }
  }*/